code/datums/skills/_skill.dm

```diff
@@ -1,11 +1,22 @@
 /datum/skill
-	var/name = "Skill"
+	var/name = "Skilling"
+	var/jobname = "Skiller"
 	var/desc = "the art of doing things"
 	var/modifiers = list(SKILL_SPEED_MODIFIER = list(1, 1, 1, 1, 1, 1, 1)) //Dictionary of modifier type - list of modifiers (indexed by level). 7 entries in each list for all 7 skill levels.
+	var/skillcapepath
+	var/list/levelUpMessages = list()
 
 /datum/skill/proc/get_skill_modifier(modifier, level)
 	return modifiers[modifier][level+1] //+1 because lists start at 1
 
+/datum/skill/New()
```

Comment:
why is this in new?

Reply:
ah i see now

Reply:
The specific redefinition makes sense to go in `New()` but not these ones.
You can perfectly put them in the list variable definition.

Reply:
@Rohesie could you explain what you mean by this

Reply:
Where you define the varaible, in `var/list/levelUpMessages = list()`, you can substitute `list()` for what is being defined in this proc.